OutOfSchool/OutOfSchool.Tests/OrganizationControllerTest.cs

```diff
@@ -1,33 +1,280 @@
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Moq;
 using NUnit.Framework;
 using OutOfSchool.WebApi.Controllers;
+using OutOfSchool.WebApi.Models;
 using OutOfSchool.WebApi.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
 
 namespace OutOfSchool.Tests
 {
     [TestFixture]
-    public class Tests
+    public class OrganizationControllerTest
     {
         private readonly ILogger<OrganizationController> _logger;
-        private IOrganizationService organizationService;
-        public OrganizationController CreateOrganizationController => new OrganizationController(_logger, organizationService);
-
-        public Tests()
+        private readonly Mock<IOrganizationService> organizationService;
+        private readonly OrganizationController organizationController;
+        private readonly ClaimsPrincipal user;
+        public OrganizationControllerTest()
         {
             _logger = new Mock<ILogger<OrganizationController>>().Object;
-            organizationService = new Mock<IOrganizationService>().Object;
+            organizationService = new Mock<IOrganizationService>();
+            organizationController = new OrganizationController(_logger, organizationService.Object);
+            user = new ClaimsPrincipal(new ClaimsIdentity());
+            organizationController.ControllerContext.HttpContext = new DefaultHttpContext { User = user };
+        }
+     
+        [Test]
+        public async Task Get_Organizations_ReturnsOkObjectResult()
+        {
+            // Arrange
+            IEnumerable<OrganizationDTO> organizations = new List<OrganizationDTO>
+            {
+                new OrganizationDTO
+                {
+                      Id = 0b101,
+                      Title = "Title",
+                      Website = "Website",
+                      Facebook = "�������",
+                      Instagram = "���������",
+                      Description = "ererer",
+                      MFO = "123456",
+                      EDRPOU = "12345678",
+                      INPP = "1234567891",
+                      Type = 0,
+                      UserId = "123",
+                },
+            }.ToList();
+
+            organizationService.Setup( it => it.GetAll()).Returns(Task.FromResult(organizations));
+
+            // Act
+            var response = await organizationController.GetOrganizations();
+
+            // Assert
+            Assert.IsInstanceOf<OkObjectResult>(response);
```

Comment:
it's nice to test not only the response type but content as well

Reply:
You said that we have to test only status codes in controller, didn't you ?

Reply:
you are right, status code is enough here